### ISSUE TITLE
loss value

### ISSUE THREAD

**Issue by ciande-narbit** [PeterWang512/GANSketching#24]
Follow the script you provided, my training losses fluctuate wildly, unable to convergence. Will your training losses be significantly reduced?

**Comment by PeterWang512** [comment]
Do you mind sharing which script are you using to train the model? Thanks!

**Comment by ciande-narbit** [comment]
train_teaser_cat.sh
with loss below
![image](https://user-images.githubusercontent.com/62127911/185068669-93b63c56-c308-4ff6-be22-44c2d48f6ca6.png)

**Comment by ciande-narbit** [comment]
Here is the content of script

#!/bin/bash
python train.py \
--name teaser_cat_augment --batch 4 \
--dataroot_sketch ./data/sketch/by_author/cat \
--dataroot_image ./data/image/cat --l_image 0.7 \
--g_pretrained ./pretrained/stylegan2-cat/netG.pth \
--d_pretrained ./pretrained/stylegan2-cat/netD.pth \
--max_iter 150000 --disable_eval --diffaug_policy translation \
